<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogParameter</class>
 <widget class="QDialog" name="DialogParameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>参数</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>6</y>
     <width>461</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tableView">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Highlight">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>49</red>
              <green>106</green>
              <blue>197</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Highlight">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>63</red>
              <green>140</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Highlight">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>49</red>
              <green>106</green>
              <blue>197</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ContiguousSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>64</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>24</number>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btGetParameter">
          <property name="text">
           <string>读RAM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btSetParameter">
          <property name="text">
           <string>写入RAM</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btClear">
          <property name="text">
           <string>清屏</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btWriteFlash">
          <property name="toolTip">
           <string>Flash写入次数有限，建议合并多次修改，一起写入</string>
          </property>
          <property name="text">
           <string>写入Flash</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btResetFlash">
          <property name="text">
           <string>重置Flash</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btShowHideExtend">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>↓</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="wExtend" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <widget class="QPushButton" name="btMixHorizontalAttitude">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>5</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>融合水平姿态</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btGetParameter</tabstop>
  <tabstop>btSetParameter</tabstop>
  <tabstop>btClear</tabstop>
  <tabstop>btWriteFlash</tabstop>
  <tabstop>btResetFlash</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
